content/zh/docs/tasks/run-application/run-single-instance-stateful-application.md

```diff
@@ -101,77 +81,57 @@ kubectl create -f https://k8s.io/docs/tasks/run-application/gce-volume.yaml
             Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
             ClaimName:  mysql-pv-claim
             ReadOnly:   false
-        Conditions:
-          Type          Status  Reason
-          ----          ------  ------
-          Available     False   MinimumReplicasUnavailable
-          Progressing   True    ReplicaSetUpdated
-        OldReplicaSets:       <none>
-        NewReplicaSet:        mysql-63082529 (1/1 replicas created)
-        Events:
-          FirstSeen    LastSeen    Count    From                SubobjectPath    Type        Reason            Message
-          ---------    --------    -----    ----                -------------    --------    ------            -------
-          33s          33s         1        {deployment-controller }             Normal      ScalingReplicaSet Scaled up replica set mysql-63082529 to 1
+            Conditions:
```

Comment:
`Conditions`, `OldReplicaSets`, `NewReplicaSet`, on the same level `Pod Template`